test/extended/networking/livemigration.go

```diff
@@ -462,3 +477,80 @@ func checkEastWestTraffic(virtClient *kubevirt.Client, vmiName string, podIPsByN
 		}
 	}
 }
+
+func isNetworkProvisioned(oc *exutil.CLI, nodeName string, networkName string) (bool, error) {
+	var podName string
+	var out string
+	var err error
+
+	out, err = runOcWithRetry(oc.AsAdmin(), "get",
+		"pods",
+		"-o", "name",
+		"-n", "openshift-ovn-kubernetes",
+		"--field-selector", fmt.Sprintf("spec.nodeName=%s", nodeName),
+		"-l", "app=ovnkube-node")
+	if err != nil {
+		return false, err
+	}
+	outReader := bufio.NewScanner(strings.NewReader(out))
+	re := regexp.MustCompile("^pod/(.*)")
+	for outReader.Scan() {
+		match := re.FindSubmatch([]byte(outReader.Text()))
+		if len(match) != 2 {
+			continue
+		}
+		podName = string(match[1])
+		break
+	}
+	if podName == "" {
+		return false, fmt.Errorf("could not find a valid ovnkube-node pod on node '%s'", nodeName)
+	}
```

Comment:
It seems this code looks for ovnkube-node pod name, cant it be done using the programmatic go-client (similar to line 508)?

Reply:
It can, but I just copied this code from somewhere else - I still need to make this coalesce.

And still, I guess staff engineering opposes this PR: https://github.com/openshift/origin/pull/29348#issuecomment-2532208445

Reply:
Refactored the code. At least now it's clear I'm re-using existent code that can be improved on a follow-up.